roles/openshift_aws/tasks/accept_nodes.yml

```diff
@@ -36,4 +38,4 @@
     nodes: "{{ instancesout.instances|map(attribute='private_dns_name') | list  }}"
     timeout: 60
   register: nodeout
-  delegate_to: "{{ mastersout.instances[0].public_ip_address }}"
+  delegate_to: "{{ groups.masters.0 }}"
```

Comment:
This is similar to https://github.com/openshift/openshift-ansible/pull/6531. I don't have direct root access to the instances I'm testing with.